packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -146,14 +144,14 @@ describe('holocronModule', () => {
         loadModuleData: fakeLoadModuleData,
         WrappedComponent: FakeComponent,
         frozenProps: fakeProps,
-        loadCount: 0,
+        currentLoadCount: 0,
         componentName: 'FakeComponent',
-        hocInstance: fakeInstance,
+        hocInstance: { ...fakeInstance },
       });
-      expect(fakeSetState).toHaveBeenCalledWith({ status: 'error' });
+      expect(fakeSetState).toHaveBeenCalledWith('error');
     });
 
-    it('should not call setState if unmounted on failure', async () => {
+    it('should not call setStatus if unmounted on failure', async () => {
```

Comment:
If the loadModuleData is throwing an error, shouldn't this be caught by the try catch, and it call `setState` with `error` ?

Reply:
In previous implementation, error message was printed on console (`console.error`) and state  was updated to `error` string in catch block. This is same implementation. I think this is more for developer, so console output should be good enough.